aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -254,6 +259,89 @@ def parse_lambda_tags_from_arn(arn):
     ]
 
 
+MEMORY_ALLOCATED_RECORD_KEY = "memorySizeMB"
+INIT_DURATION_RECORD_KEY = "initDurationMs"
+BILLED_DURATION_RECORD_KEY = "billedDurationMs"
+RUNTIME_METRICS_BY_RECORD_KEY = {
+    # Except INIT_DURATION_RECORD_KEY which is handled separately
+    "durationMs": DURATION_METRIC_NAME,
+    BILLED_DURATION_RECORD_KEY: BILLED_DURATION_METRIC_NAME,
+    "maxMemoryUsedMB": MAX_MEMORY_USED_METRIC_NAME,
+}
```

Comment:
nit: could you move these to the top of the file.
nit: let's put `dureationMs` and `maxMemeoryUsedMB` as keys as well